### ISSUE TITLE
Support bboxes in Cityscapes

### ISSUE THREAD

**Issue by MohsinMohammad** [openvinotoolkit/datumaro#480]
[In](url) the dataset(Cityscapes1) folder, I have placed below folders 

- imgsFine
    - leftImg8bit
         - train
             - <city>
                 - *_leftImg8bit.png
- gtFine
- gtBboxCityPersons
- gtBbox3d
- gtCoarse

and then executed below commands
$ datum create
$ datum import -f cityscapes -i D:\Downloads\Compressed\Cityscapes1 --overwrite
$ datum export -f voc -o city_out --overwrite

But the generated VOC XML files do not contain object tag.

**Comment by MohsinMohammad** [comment]
![xml_content](https://user-images.githubusercontent.com/19344353/135569680-7cba0239-3928-4d5c-a96e-f70a3ebf20dd.PNG)

**Comment by zhiltsov-max** [comment]
Hi, Currently, our Cityscapes format implementation [only works with segmentation masks](https://openvinotoolkit.github.io/datumaro/docs/formats/cityscapes/#import-cityscapes-dataset). We're open for requests and PRs on boxes and cuboids.

**Comment by MohsinMohammad** [comment]
Thanks @zhiltsov-max . With your solution, able to get the bbox info in XML file.
In the generated XML file, all labels are included. However, I need only few labels ( person, car, truck). 

I have tried below commands

```
datum create <...>
datum import <...>
datum transform -t shapes_to_boxes -o boxes_proj
datum transform -p boxes_proj -t remap_labels -o proj_new -- -l person:person -l car:car -l truck:truck -l bus:bus --default delete
datum export -p proj_new -f voc -o cityscapes -- --save-images
```
When export command is run, below warning is shown.
_WARNING: The following labels are remapped to background: car, truck, bus_

Then in the XML file, only **person** label is present.
Can you let me know how I can retain the required few labels info in the XML

**Comment by zhiltsov-max** [comment]
@MohsinMohammad, apparently, you've met the issue being fixed in #478. In the version you're using, try to call export with the `-- --label-map cityscapes` parameter (if the labels are from Cityscapes) or create a colormap file like this (for custom labels):

labels.txt
```
# B G R label_name
0 0 0 background
1 2 1 person
4 5 6 car
7 8 9 truck
5 12 6 bus
...
```
And pass it as `-- --label-map labels.txt`.
